src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -284,9 +293,13 @@ public virtual string NormalizeKey(string key)
         public virtual async Task<IdentityResult> SetRoleNameAsync(TRole role, string name)
         {
             ThrowIfDisposed();
-            await Store.SetRoleNameAsync(role, name, CancellationToken);
-            await UpdateNormalizedRoleNameAsync(role);
-            return await LogResultAsync(IdentityResult.Success, role);
+
+            using (await BeginLogScopeAsync(role))
```

Comment:
BeginLogScope to me reads a but weird. Logging instead of Log would be my preference, but English isn't my mother's tongue.